buildSrc/src/main/kotlin/eternalcode.java-conventions.gradle.kts

```
@@ -1,10 +1,20 @@
 plugins {
     `java-library`
+    checkstyle
 }
 
 group = "com.eternalcode"
 version = "1.0.0"
 
+checkstyle {
+    toolVersion = "10.6.0"
+
+    configFile = file("${rootDir}/config/checkstyle/checkstyle.xml")
+
+    maxErrors = 0
+    maxWarnings = 0
+}
+
 repositories {
     mavenLocal()
     maven { url = uri("https://hub.spigotmc.org/nexus/content/repositories/snapshots/") }
```

```suggestion
    maven("https://hub.spigotmc.org/nexus/content/repositories/snapshots/")
```

Czy na pewno potrzebujemy rozszerzenia `{url = uri() }`?

---

To jest .kts

---

akurat model DSL nie ma nic do tego, bez tego rozszerzenia to normalnie działa.

---

Ignoruje, brak uzasadnienia dlaczego zmiana powinna być wprowadzona, obecnie zostawiam pełną wersje repository